server/game/cards/05-DT/AnkylosaurusFormation.js

```diff
@@ -0,0 +1,38 @@
+const Card = require('../../Card.js');
+
+class AnkylosaurusFormation extends Card {
+    //Play: Choose one:
+    //• A friendly creature gains skirmish for the rest of your turn.
+    //• Exalt a friendly creature. Each friendly creature gains skirmish for the rest of your turn.
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
```

Comment:
This card is a choose. Need to use mode select.

Reply:
i started on that, but then though this way would eliminate a UX step.  One creature I always going to be targeted to get skirmish, the choice is really do you wanna exalt it or not for the larger effect

Right now its Choose creature, exalt yes/no

Moving it to the mode select would be choose skirmish/ or exalt  then target a creature. ...i guess thats the same amount of clicking.  

I can switch it if ya want

Reply:
That's tricky 👍